src/ui.cpp

```diff
@@ -607,15 +613,15 @@ void uilist::calc_data()
                             realtextwidth = w;
                         }
                     }
-                    if( realtextwidth + 4 > w_width ) {
-                        w_width = realtextwidth + 4;
+                    if( realtextwidth + WIDTH_BUFFER > w_width ) {
+                        w_width = realtextwidth + WIDTH_BUFFER;
                     }
                 }
             }
         } else if( textwidth != -1 ) {
             realtextwidth = textwidth;
-            if( realtextwidth + 4 > w_width ) {
-                w_width = realtextwidth + 4;
+            if( realtextwidth + WIDTH_BUFFER > w_width ) {
+                w_width = realtextwidth + WIDTH_BUFFER;
```

Comment:
I think `text` is printed before all uilsit entries and does not start with a hotkey hint, so changing 4 to `WIDTH_BUFFER` is not needed.